source/features/prevent-link-loss.tsx

```diff
@@ -53,28 +53,29 @@ function isVulnerableToLinkLoss(value: string): boolean {
 		|| value !== value.replace(discussionUrlRegex, preventDiscussionLinkLoss);
 }
 
-const updateUI = debounceFn(({delegateTarget: field}: DelegateEvent<Event, HTMLTextAreaElement>): void => {
-	if (!isVulnerableToLinkLoss(field.value)) {
-		getUI(field).remove();
-	} else if (pageDetect.isNewIssue() || pageDetect.isNewRelease() || pageDetect.isCompare()) {
-		$('file-attachment', field.form!)!.append(
-			getUI(field, 'mt-2', 'mx-0', 'mx-md-2'),
-		);
+function updateUI({delegateTarget: field}: DelegateEvent<Event, HTMLTextAreaElement>): void {
+	if (isVulnerableToLinkLoss(field.value)) {
+		$('file-attachment .js-write-bucket', field.form!)!.append(getUI(field));
 	} else {
-		$('.form-actions', field.form!)!.before(
-			getUI(field, 'mx-md-2', 'mb-2'),
```

Comment:
Has this difference been eliminated now? We still have two different forms, are they both covered by the new selector?

Reply:
> Has this difference been eliminated now?

Looks like it, at least it works fine in all the forms I've tested with.